packages/eslint-plugin-ecmascript-compat/lib/rule.js

```diff
@@ -3,9 +3,6 @@ const { createDelegatee, delegatingVisitor } = require('./delegation');
 const features = require('./features');
 const targetRuntimes = require('./targetRuntimes');
 
-const targets = targetRuntimes();
```

Comment:
> I've been trying it out - One problem is that the console logs now happen once per file.
>
> I could memoize getting the target rules, but then it still might flip-flop depending on the size of the memoize cache.
>
> any suggestions?

IIRC ESLint doesn't have a rule lifecycle beyond that of a file unfortunately.

I think there's some big work/rewrite planned for ESLint, maybe this is something they'd address.

In the meantime perhaps some of the more complex lint plugins have some ideas we could use.

It does seem like something that needs to be solved before this goes in.

Reply:
Actually IIRC #76 concluded with that we shouldn't be console logging anyway

Reply:
So remove all console logging and this is resolved ?

Reply:
Yes that should be fine

Reply:
I also removed a console.warn and mentioned in the readme instead, so this PR will also resolve #76